src/LogzioShipper/text_parser.py

```diff
@@ -0,0 +1,15 @@
+from typing import Generator
+from .file_parser import FileParser
+
+
+class TextParser(FileParser):
+
+    def __init__(self, file_data: str) -> None:
+        super().__init__(file_data)
+
+    def parse_file(self) -> Generator:
```

Comment:
In the next version we should add support to multiline

Reply:
did multiline :)